src/Capsule.cc

```diff
@@ -180,6 +180,14 @@ std::optional<gz::math::Inertiald> Capsule::CalculateInertial(double _density)
   }
 }
 
+/////////////////////////////////////////////////
+gz::math::AxisAlignedBox Capsule::AxisAlignedBox() const
+{
+  auto halfSize = gz::math::Vector3d(
+    this->Radius(), this->Radius(), this->Length() / 2);
```

Comment:
```suggestion
    this->Radius(), this->Radius(), this->Radius() + this->Length() / 2);
```

I think the AABB for a capsule needs to be bigger than a cylinder; currently they are the same

if my suggestion is correct, you may consider other ways of expressing it as well (up to you): `this->Radius() * gz::math::Vector3d::One + this->Length / 2 * gz::math::Vector3d::UnitZ`

Reply:
Oops, my mistake. Thanks for catching this, I have fixed it using the second suggestion in https://github.com/gazebosim/sdformat/pull/1547/commits/6f3f7387f2e8da7533f53b422bec0273413908c9 because I think it makes it clear that the capsule is a sphere with an added length along Z.